src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java

```diff
@@ -0,0 +1,24 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_Interstital() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)